contracts/staking/StakingInfo.sol

```diff
@@ -100,13 +100,13 @@ contract StakingInfo {
         address indexed newSigner,
         bytes signerPubkey
     );
-    event ReStaked(uint256 indexed validatorId, uint256 amount, uint256 total);
+    event Restaked(uint256 indexed validatorId, uint256 amount, uint256 total);
     event Jailed(
         uint256 indexed validatorId,
         uint256 indexed exitEpoch,
         address indexed signer
     );
-    event UnJailed(uint256 indexed validatorId, address indexed signer);
+    event Unjailed(uint256 indexed validatorId, address indexed signer);
```

Comment:
It would be great if we could use `UnJailed` only because Heimdall uses this event.

Reply:
Other than that looks good.

Reply:
alright